### PR TITLE
refactor: drop edit provider

### PR DESCRIPTION
**Pull request by Kneesal**
# Description

### Issue

we want to make UI editable always

[Link to Basecamp Todo](https://3.basecamp.com/3105655/projects)

### Solution

drop edit provider to enforce permanent edit state

### CODE REVIEW

File: apps/videos-admin/src/app/[locale]/(dashboard)/videos/[videoId]/_VideoView/Metadata/VideoDescription/VideoDescription.tsx [hunk 1]
```diff
@@ -75,15 +71,13 @@ export function VideoDescription({
...
-              disabled={!isEdit}
               minRows={6}
               maxRows={6}
             />
-            {isEdit && (
-              <Stack direction="row" justifyContent="flex-end">
-                <SaveButton disabled={!isValid || isSubmitting || !dirty} />
-              </Stack>
-            )}
+
```
Comment (edmonday) [inline apps/videos-admin/src/app/[locale]/(dashboard)/videos/[videoId]/_VideoView/Metadata/VideoDescription/VideoDescription.tsx:77]:
```suggestion
```
NIT: remove empty line

File: apps/videos-admin/src/app/[locale]/(dashboard)/videos/[videoId]/_VideoView/Metadata/VideoSnippet/VideoSnippet.tsx [hunk 2]
```diff
@@ -71,15 +67,13 @@ export function VideoSnippet({
...
-              disabled={!isEdit}
               minRows={6}
               maxRows={6}
             />
-            {isEdit && (
-              <Stack direction="row" justifyContent="flex-end">
-                <SaveButton disabled={!isValid || isSubmitting || !dirty} />
-              </Stack>
-            )}
+
```
Comment (edmonday) [inline apps/videos-admin/src/app/[locale]/(dashboard)/videos/[videoId]/_VideoView/Metadata/VideoSnippet/VideoSnippet.tsx:73]:
```suggestion
```
NIT: remove line

File: apps/videos-admin/src/app/[locale]/(dashboard)/videos/[videoId]/_VideoView/Children/Children.tsx [hunk 3]
```diff
@@ -82,14 +78,6 @@ export function Children({ childVideos }: ChildrenProps): ReactElement {
   return (
     <Section
       title={t('Children')}
-      action={
```
Comment (edmonday) [inline apps/videos-admin/src/app/[locale]/(dashboard)/videos/[videoId]/_VideoView/Children/Children.tsx:85]: This going to get moved? The create child button?

**Review by edmonday — CHANGES_REQUESTED** [review]

File: apps/videos-admin/src/app/[locale]/(dashboard)/videos/[videoId]/_VideoView/Children/Children.tsx (hunk 3 above)
Comment (Kneesal) [reply apps/videos-admin/src/app/[locale]/(dashboard)/videos/[videoId]/_VideoView/Children/Children.tsx:85]: yep

**Review by edmonday — APPROVED** [review]

**Pull request merged**